### DISCUSSION TITLE
unknown option '-install' on windows 11

### DISCUSSION THREAD

**Discussion by nbblk** [FiloSottile/mkcert#437] [Q&A] [answered]
Hello.
I just installed mkcert via chocolatey on powershell admin, but it doesn't show -install option. I reinstalled the package and tried cmd and git bash but still the same. 
The versions of mkcert / chocolatey / windows are like below. What can I do to solve this issue? Thanks in advance.
![image](https://user-images.githubusercontent.com/75354780/149606461-8bbe683b-f5d3-473d-ba39-7a2bd4ecd599.png)

**Comment by FiloSottile** [comment, marked as answer]
Looks like a different tool called mkcert is earlier in the `%PATH%`.

**Comment by SeojinSeojin** [comment]
Maybe you installed mkcert in your global package.json.

https://github.com/Subash/mkcert
